Week2/Bank.java

```diff
@@ -0,0 +1,118 @@
+package Week2;
+
+import java.util.*;
+import java.math.BigDecimal;
+//  계좌생성, 출금, 입금, 송금 메서드 구현
+// 내부에서 입력을 받는 액션?
+// 적금 계좌는
+public class Bank {
+    private ArrayList<Account> accountArray;
+    private HashMap<AccountType, InterestCalculator> interestCalculator;
+
+    public Bank(){
+        this.accountArray = new ArrayList<>();
+        this.interestCalculator = new HashMap<>();
+        this.interestCalculator.put(AccountType.NORMAL_ACCOUNT, new NormalInterestCalculator());
+        this.interestCalculator.put(AccountType.SAVING_ACCOUNT, new SavingInterestCalculator());
+    }
+
+    // 계좌생성
+    // 한 메소드안에 NORMAL과 SAVING을 같이 넣을려했지만 매개변수 이슈로 나눔
+    public void createAccount(AccountType accountType, String accountNumber, String owner,

```

Comment:
메소드명을 더 구체적으로 작성해야 할 것 같습니다.
어떤 Account를 생성하는지 명시해줘야 합니다.
```suggestion
    public void createBasicAccount(AccountType accountType, String accountNumber, String owner,

```